### DISCUSSION TITLE
Remove "Add to Calendar"

### DISCUSSION THREAD

**Discussion by ACAwebbuilder** [code4recovery/12-step-meeting-list#1245] [Q&A] [answered]
Hi!
Does anyone know of a way to remove the "Add to Calendar" button from the meeting page?

**Comment by anchovie91471** [comment]
We don’t really have an easy way to do that. What is your reason for wanting it to be removed?

Would you mind posting a link to the site as well so I can have a look?

**Comment by ACAwebbuilder** [comment]
Hi Anthony!  The meeting are listed on this page
https://rvaacadf.org/meetings/.

The intergroup asked if I could take it off.  Not sure why they don't want
it on there.

**Comment by anchovie91471** [comment, marked as answer]
We try to present a consistent user interface and experience across sites that use TSML so there isn't an easy way to remove the button. If you can find out from the intergroup what issues they might be having that causes them to want to remove a feature, we can consider adding a setting or fix the issue.

Otherwise, the only way to remove it is via a CSS rule that simply hides it (note, this could cause future features to be hidden so use at your own risk):

```
#tsml-ui div div div div button {
    display: none;
} 
```
Again, we would prefer to better know the needs/issues underlying the request and fix them with in a more reliable and predictable way.

Thanks!
